Podfile

```diff
@@ -8,4 +8,14 @@ target 'ITEvents' do
   pod 'Reusable'
   pod 'TagListView'
 
+  
+  post_install do |installer|
+      installer.pods_project.targets.each do |target|
+          if target.name == 'SwinjectStoryboard' || 'Reusable'
```

Comment:
here is a bit better way of writing this:
https://github.com/CocoaPods/CocoaPods/issues/6791#issuecomment-329331953